CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to this repository
+
+## Contributions in English
+
+### Using Bold
+- to note step numbers. Example: __First__, __Second__, __Third__, etc
```

Comment:
I think we should use lists for steps, if contributor don't want to use list, I don't think we need to force them use bold on it. Anyway, I don't think it should be a rule

Reply:
Or use numbers for steps, for example:

1. xxxxx
2. xxxxx
3. xxxxx

Reply:
See possible examples below